src/helpers.js

```diff
@@ -179,8 +179,34 @@ exports.allowedZero = function(val, char, caretPos, options) {
  * @param {val} string value to convert to a number
  * @param {options} Finput options object
  */
-exports.toNumber = function(val, options) {
-  return val && Number(val.replace(new RegExp(`[${options.thousands}]`, 'g'), ''));
+exports.toNumber = function(stringValue, options) {
+  if (!stringValue) return null;
```

Comment:
We should consider renaming these toNumber / toString methods to something more suitable if they're not going to always parse into a Number / String.